src/apps/companies/apps/edit-history/client/EditHistoryList.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import DateUtils from 'data-hub-components/dist/utils/DateUtils'
+import Pagination from 'data-hub-components/dist/pagination/Pagination'
```

Comment:
This can be changed now:
```suggestion
import { DateUtils, Pagination } from 'data-hub-components'
```

Reply:
👍